presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -695,6 +695,12 @@ public HiveInsertTableHandle beginInsert(ConnectorSession session, ConnectorTabl
 
         List<HiveColumnHandle> handles = hiveColumnHandles(connectorId, table.get());
 
+        for (HiveColumnHandle hiveColumnHandle : handles) {
+            if (!HiveWriteUtils.isWritableType(hiveColumnHandle.getHiveType())) {
+                throw new PrestoException(NOT_SUPPORTED, format("Inserting into Hive table with column type %s not supported.", hiveColumnHandle.getHiveType()));
```

Comment:
How about wording this is

"The Hive connector does not support writing the column type $TYPE"